payments-core/src/main/java/com/stripe/android/model/PaymentMethodCreateParams.kt

```diff
@@ -44,7 +52,7 @@ data class PaymentMethodCreateParams internal constructor(
         @JvmSynthetic
         get() {
             return when (type) {
-                Type.Card -> card?.attribution?.plus(productUsage)
+                Type.Card -> (card?.attribution ?: emptySet()).plus(productUsage)
```

Comment:
Tracking issues with missing `product_usage` in MOBILESDK-306, but here if the card didn't have an `attribution` we wouldn't include `productUsage` either.